### DISCUSSION TITLE
Lvar - not working?

### DISCUSSION THREAD

**Discussion by LukasGNN** [flybywiresim/aircraft#3701] [Help]
How to trigger LS button? i have only Lvar but this does not update info on PFD, only light up the button..If i press manually i get ILS aligment markers, but with Lvar i get nothing. The same actually happen to CSTR/VOR/WPT/NDB/ARPT buttons on FCU (only light, no info on PFD).  No action with Lvar change on Eng1 and Eng2 Bleed , Hot Air ether... ANy suggestions how to get it to work?

**Comment by tomber42** [comment]
What is LVAR?
